schematic/models/validate_attribute.py

```diff
@@ -378,11 +385,148 @@ def generate_content_error(
             error_row=error_row,
             error_col=attribute_name,
             error_message=error_message,
-            error_val=error_val,
+            error_val=invalid_entry,
         )
 
         return error_list, warning_list
 
+    def _get_rule_attributes(
+        val_rule: str, error_col_name: str, dmge: DataModelGraphExplorer
+    ) -> tuple[[str, str, str]]:
```

Comment:
The return statement is returnign 6 things, but this shows only 3. Also the double brackets are uneeded.